lix/packages/client/src/git/findRepoRoot.ts

```diff
@@ -0,0 +1,14 @@
+import type { NodeishFilesystem } from "@lix-js/fs"
+import { findRoot } from "../../vendored/isomorphic-git/index.js"
+
+export async function findRepoRoot(args: {
+	nodeishFs: NodeishFilesystem
+	path: string
+}): Promise<string | undefined> {
+	const gitroot = await findRoot({
+		fs: args.nodeishFs,
+		filepath: args.path,
+	}).catch(() => undefined)
+
+	return gitroot ? "file://" + gitroot : undefined
```

Comment:
Why do we add file:// to the path here? Guess this is not a change in this pr but we might add a TODO to document the reason

Reply:
because openRepo expects a valid lix url  this is needed to pass the return value of findRepoRoot to open repo:

openRepository(await findRepoRoot ....


please explain why this was not clear as i am not sure what a helpful comment would be